modules/light-clients/08-wasm/types/events.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	"fmt"
+
+	ibcexported "github.com/cosmos/ibc-go/v7/modules/core/exported"
+)
+
+// IBC 08-wasm events
+const (
+	EventTypeStoreWasmCode = "store_wasm_code"
+
+	AttributeKeyWasmCodeHash = "wasm_code_hash"
+)
+
+var AttributeValueCategory = fmt.Sprintf("%s_%s", ibcexported.ModuleName, ModuleName)
```

Comment:
linter might complain because of no godoc on exported symbol, but I'm not 100% sure.